### PR TITLE
Add Missing Tests for Autocomplete

### PR DESCRIPTION
**Pull request by SuhailLoya**

### CODE REVIEW

File: src/test/java/careconnect/logic/autocompleter/AutocompleterTest.java [hunk 1]
```diff
@@ -42,5 +42,7 @@ public void autocompleteWithLexicalPriority_validValues_correctResults() throws
         list.add("add");
         list.add("adda");
         assertEquals("add", autocompleter.autocompleteWithLexicalPriority("ad", list));
+        list.add("abs");
+        assertEquals("abs", autocompleter.autocompleteWithLexicalPriority("a", list));
```
Comment (AppleJem) [inline src/test/java/careconnect/logic/autocompleter/AutocompleterTest.java:46]: Great addition of edge case

File: src/main/java/careconnect/logic/autocompleter/exceptions/AutocompleteException.java [hunk 2]
```diff
@@ -10,7 +10,7 @@ public class AutocompleteException extends IllegalValueException {
     public AutocompleteException(String message) {
         super(message);
     }
-
+    // This constructore is added for consistency sake, currently is not being utilized.
```
Comment (AppleJem) [inline src/main/java/careconnect/logic/autocompleter/exceptions/AutocompleteException.java:13]: typo!

Comment (SuhailLoya) [reply src/main/java/careconnect/logic/autocompleter/exceptions/AutocompleteException.java]: thanks corrected!

**Review by AppleJem — APPROVED** [review]
